downstream/modules/platform/ref-controller-other-search-considerations.adoc

```diff
@@ -0,0 +1,14 @@
+[id="ref-controller-other-search-considerations"]
+
+= Other search considerations
+
+Be aware of the following issues when searching in {ControllerName}:
+
+* There is currently no supported syntax for *OR* queries. 
+All search terms are *AND*ed in the query parameters.
+* The left-hand portion of a search parameter can be wrapped in quotes
+to support searching for strings with spaces. For more information, see xref:ref-controller-search-tips[Tips for searching].
+* Currently, the values in the Fields are direct attributes expected to be returned in a *GET* request. 
+Whenever you search against one of the values, {ControllerName} carries out an `__icontains` search. 
+So, for example, `name:localhost` sends back `+?name__icontains=localhost+`.
+{ControllerNameStart} currently performs this search for every Field value, even `id`, which is not ideal.
```

Comment:
I might leave out "which is not ideal" here as it has a casual and negative connotation.